src/quicktype-core/Type.ts

```diff
@@ -479,8 +479,8 @@ export class ObjectType extends Type {
         }
     }
 
-    setProperties(properties: ReadonlyMap<string, ClassProperty>, additionalPropertiesRef: TypeRef | undefined) {
-        assert(this._properties === undefined, "Tried to set object properties twice");
+    setProperties(properties: ReadonlyMap<string, ClassProperty>, additionalPropertiesRef: TypeRef | undefined, assertFirstTime = true) {
+      if (assertFirstTime) assert(this._properties === undefined, "Tried to set object properties twice");
```

Comment:
I did this to be able to change properties order

Reply:
In python default properties must be last

Reply:
@schani will have to approve